From b7e78afe297e0194c22d7901bfcc34b984659165 Mon Sep 17 00:00:00 2001
From: Michal Domonkos <mdomonko@redhat.com>
Date: Thu, 2 Oct 2025 10:38:13 +0200
Subject: [PATCH] Fix broken redirects to man pages

When the site is deployed, whether on rpm-software-management.github.io
for the upstream version or rpm.org for the stable versions, its root is
never /, it's either /rpm or /docs/<version>, respectively. So no wonder
the absolute /man/... redirects never worked in either case...

They did work when rendering the site locally with "make site" where the
site's root is /, though, which is why I happily used that in commit
ca6a6b1b40acb2e48b5c6eb337676f9c7d18d44d, with the same pattern being
repeated (naturally) for later man page conversions as well.

Fix by simply using a relative path instead. This also makes sure we
actually redirect to the correct version of the man page on rpm.org.

Fixes: #3964
(backported from commit c25d3530f6b0a7231be78412f38eb8f2283cb241)
---
 docs/manual/lua.md         | 2 +-
 docs/manual/macros.md      | 2 +-
 docs/manual/queryformat.md | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/docs/manual/lua.md b/docs/manual/lua.md
index 898376f4ae..904a3c219b 100644
--- a/docs/manual/lua.md
+++ b/docs/manual/lua.md
@@ -1,5 +1,5 @@
 ---
 layout: redirected
 sitemap: false
-redirect_to: /man/rpm-lua.7
+redirect_to: ../man/rpm-lua.7
 ---
diff --git a/docs/manual/macros.md b/docs/manual/macros.md
index 1bb0155507..a736c2db40 100644
--- a/docs/manual/macros.md
+++ b/docs/manual/macros.md
@@ -1,5 +1,5 @@
 ---
 layout: redirected
 sitemap: false
-redirect_to: /man/rpm-macros.7
+redirect_to: ../man/rpm-macros.7
 ---
diff --git a/docs/manual/queryformat.md b/docs/manual/queryformat.md
index ae8013a81f..5f4913ffdd 100644
--- a/docs/manual/queryformat.md
+++ b/docs/manual/queryformat.md
@@ -1,5 +1,5 @@
 ---
 layout: redirected
 sitemap: false
-redirect_to: /man/rpm-queryformat.7
+redirect_to: ../man/rpm-queryformat.7
 ---
